tor/net.go

```diff
@@ -42,7 +56,9 @@ func (r *ClearNet) LookupHost(host string) ([]string, error) {
 }
 
 // LookupSRV for regular network uses net.LookupSRV function
-func (r *ClearNet) LookupSRV(service, proto, name string) (string, []*net.SRV, error) {
+func (r *ClearNet) LookupSRV(service, proto, name string,
+	_ time.Duration) (string, []*net.SRV, error) {
```

Comment:
Thoughts on also using the timeout here? We can add a context deadline to `net.DefaultResolver.LookupSRV`. which is what `net.LookupSRV` invokes.

Reply:
Sure, wasn't aware of that😂
Timeout context added.